code/obj/machinery/computer.dm

```diff
@@ -12,19 +12,77 @@
 	/// does it have a glow in the dark screen? see computer_screens.dmi
 	var/glow_in_dark_screen = TRUE
 	var/image/screen_image
+
+	var/can_reconnect = 0 //Set to 1 to make multitools call connection_scan. For consoles with associated equipment (cloner, genetek etc)
+	var/obj/item/circuitboard/circuit_type = null
+	Topic(href, href_list)
+		if (..(href, href_list))
+			return 1
+		playsound(src.loc, 'sound/machines/keypress.ogg', 30, 1, -15)
+
+	attack_hand(var/mob/user)
+		..()
+		if (!user.literate)
+			boutput(user, "<span class='alert'>You don't know how to read or write, operating a computer isn't going to work!</span>")
+			return 1
+		interact_particle(user,src)
```

Comment:
Isn't part of previous implementation, AFAICT from diff on GitHub without looking at the files any deeper.

Reply:
You should find the original implementation as the last diff and only removal on this file. However, it may have not been working as the proc was overwritten due to computer being defined again in banking.dm. I'll need to check which overwrote which, but this may be reactivating a feature that was lost at some point. I can then remove the change or add it to the PR description.

Reply:
I did some testing to compare the feature branch to the current master. Illiteracy prevents using computers currently. So the proc was not overwritten and it is not functionally changed in the feature. However, there are some bugs with TGUI computers creating dialogs even if you are illiterate. This happens in both the current master and feature branches. I'll look into fixing that in a future PR.